src/Core/Prefab/PlanarView.js

```diff
@@ -0,0 +1,262 @@
+
+    this.initDefaultProviders(scheduler);
+
+    // Configure camera
+    const dim = boundingbox.dimensions();
+    const positionCamera = boundingbox.center().clone();
+    positionCamera._values[2] = Math.max(dim.x, dim.y);
+    const lookat = positionCamera.xyz();
+    lookat.z = 0;
+
+    this.camera.setPosition(positionCamera.xyz());
+    this.camera.camera3D.lookAt(lookat);
+    this.camera.camera3D.near = 0.1;
+    this.camera.camera3D.far = 2 * Math.max(dim.x, dim.y);
+    this.camera.camera3D.updateProjectionMatrix();
+    this.camera.camera3D.updateMatrixWorld(true);
+
+
+    // Configure tiles
+    const nodeInitFn = function nodeInitFn(context, layer, parent, node) {
+        node.materials[0].setLightingOn(layer.lighting.enable);
+        node.materials[0].uniforms.lightPosition.value = layer.lighting.position;
+
+        if (__DEBUG__) {
+            node.material.uniforms.showOutline = { value: layer.showOutline || false };
+            node.material.wireframe = layer.wireframe || false;
+        }
+    };
+    const SSE_SUBDIVISION_THRESHOLD = 6.0;
+
+    const tileLayer = new GeometryLayer('planar');
+    const initLayer = initTiledGeometryLayer(planarSchemeTile(boundingbox));
+
+    tileLayer.preUpdate = (context, layer) => {
+        if (layer.level0Nodes === undefined) {
+            initLayer(context, layer);
+        }
+        return layer.level0Nodes;
+    };
+    tileLayer.update =
+        processTiledGeometryNode(
+            planarCulling,
+            planarSubdivisionControl(6, SSE_SUBDIVISION_THRESHOLD),
+            nodeInitFn);
+    tileLayer.builder = new PlanarTileBuilder();
+
+    const threejsLayer = engine.getUniqueThreejsLayer();
+    tileLayer.type = 'geometry';
+    tileLayer.protocol = 'tile';
+    tileLayer.threejsLayer = threejsLayer;
+    tileLayer.lighting = {
+        enable: false,
+        position: { x: -0.5, y: 0.0, z: 1.0 },
+    };
+    this.camera.camera3D.layers.enable(threejsLayer);
+
+    this.addLayer(tileLayer);
+
+
+    /*
+    this.controls = new GlobeControls(
+        this.camera.camera3D,
+        positionTargetCamera.as('EPSG:4978').xyz(),
+        engine.renderer.domElement,
+        engine,
+        size,
+        'EPSG:4978',
+        this.getPickingPositionFromDepth.bind(this));
+    this.controls.rotateSpeed = 0.25;
+    this.controls.zoomSpeed = 2.0;
+    this.controls.minDistance = 30;
+    this.controls.maxDistance = size * 8.0;
+
+    this.controls.addEventListener('change', () => {
+        this.camera.update();
+        this.notifyChange(0, true);
+    });*/
+
+    // this.controls.addEventListener('selectClick', (event) => {
+    //     this.selectNodeAt(event.mouse);
+    // }, false);
+
+    this._renderState = RendererConstant.FINAL;
+
+    this.tileLayer = tileLayer;
+}
+
+PlanarView.prototype = Object.create(View.prototype);
+PlanarView.prototype.constructor = PlanarView;
+
+PlanarView.prototype.addLayer = function addLayer(layer) {
```

Comment:
Yes you can, using:
```js
itowns.View.prototype.addLayer.call(view, pointcloud);
```
The syntax is special, but it's an advanced use case so I don't mind it.